docs/usage.md

```diff
@@ -16,6 +16,8 @@ The tool is a command line application built with Java. See below for usage inst
 
 # Usage
```

Comment:
Perhaps we should move the section you just added to a new 'Installation' section.

Reply:
Done! ptal